docs/user_guide/setup.rst

```diff
@@ -4,128 +4,107 @@ Setup
+    ~/.errbot-ve/pip install errbot
 
 
-Extra dependencies
-^^^^^^^^^^^^^^^^^^
+If you have virtualenvwrapper installed it is even simpler::
 
-Errbot's default dependency list
-contains only the bare minimum list of dependencies
-needed to run Errbot.
-Depending on the backend you choose,
-additional requirements need to be installed.
+    mkvirtualenv -p `which python3` errbot-ve
+    pip install errbot
 
-This means that you will need to install some extra dependencies
-to make use of the backend suitable for the chat network you are using.
 
-* For XMPP servers::
+Option 3: Installing Errbot at the system level (not recommended)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Errbot may be installed directly from PyPi using `pip`_ by issuing::
 
-      pip install sleekxmpp pyasn1 pyasn1-modules
+    pip3 install errbot
 
-* For IRC servers::
 
-      pip install irc
+.. note::
+    Some of errbot's dependencies need to build C extensions
+    which means you need to have development headers for some libraries installed.
+    On Debian/Ubuntu these may be installed with
+    `apt-get install python3-dev libssl-dev libffi-dev`
+    (use `python-dev` instead of `python3-dev` if you're still stuck on Python 2).
+    Package names may differ on other OS's.
+
+.. _configuration:
 
-* For HipChat::
+First run
+^^^^^^^^^
 
-      pip install sleekxmpp pyasn1 pyasn1-modules hypchat
+You can quickly configure Errbot by first creating a working directory and calling `errbot --init`::
 
-* For Slack::
+    mkdir ~/errbot-root
+    cd ~/errbot-root
+    errbot --init
 
-      pip install slackclient
+This will create a minimally working errbot in text (development) mode. You can try it right away::
 
-* For Telegram messenger::
+    errbot
+    [...]
+    >>>
 
-      pip install python-telegram-bot
 
+`>>>` is a prompt, you can talk to errbot directly. You can try::
 
-.. _configuration:
+    !tryme
+    !help
+    !about
 
 Configuration
 -------------
 
-Once you have installed errbot,
-you will have to configure it to connect to your desired chat network.
-First, create a directory somewhere on your system
-where errbot may store its configuration and data.
-
-Once you have created the directory,
-change into it and copy the default configuration file into place.
-There are two ways to do this:
-
-1. You can generate it directly from your errbot installation with:
+Once you have installed errbot and did `errbot --init`, you will have to tweak the generated `config.py` to connect
+to your desired chat network.
 
-    `python -c "import errbot;import os;import shutil;shutil.copyfile(os.path.dirname(errbot.__file__) + os.path.sep + 'config-template.py', 'config.py')"`
+You can use https://raw.githubusercontent.com/errbotio/errbot/master/errbot/config-template.py as a base
+ for your `config.py`.
 
-2. Or you can download the :download:`config-template.py` file manually and save it as `config.py`.
```

Comment:
I think it's better to keep the `:download:` version instead of linking directly to GitHub, as this will ensure it links to the version of `config-template.py` that matches the version of the docs you're browsing.